.github/workflows/frontend-check-core-changes.yml

```
@@ -4,6 +4,12 @@ on:
     secrets:
       github-token:
         required: true
+    inputs:
+      project-path:
+        description: project path directory(defaults to 'src/**')
+        required: false
+        type: string
+        default: "src/**"
```

```suggestion
        default: "ClientApp/src/**"
```

so it is not breaking

---

applied